libraries/core_libs/consensus/include/dag/sortition_params_manager.hpp

```diff
@@ -0,0 +1,83 @@
+#pragma once
+
+#include "pbft/sync_block.hpp"
+#include "storage/storage.hpp"
+#include "vdf/config.hpp"
+
+namespace taraxa {
+
+struct SortitionParamsChange {
+  uint64_t period = 0;
+  VrfParams vrf_params;
+  uint16_t interval_efficiency = 0;
+  uint16_t actual_correction_per_percent = 1;
+
+  SortitionParamsChange() = default;
+  SortitionParamsChange(uint64_t period, uint16_t efficiency, VrfParams const& vrf);
```

Comment:
please refactor new classes and refactored functions according to const Type& styling

Reply:
It is also should be a separate PR or at least a commit if you want it for some reason it to be done in this PR. It is easier(or just not very harder) to change it in whole project then separate just my changes

Reply:
I did it just for new files and in some places in refactored code but maybe not in the all

Reply:
we agreed that we are going to follow our style for all new or refactored code :)